verify_rpms/cmd_runner.py

```diff
@@ -0,0 +1,22 @@
+from subprocess import run, CompletedProcess, CalledProcessError
+from verify_rpms.exceptions import CmdError
+
+
+class CmdRunner:
```

Comment:
I have to say I don't see a point in having this class. I don't think that:
```
foo = CmdRunner.run_cmd(["foo", "bar", "baz"])
```
Is a very significant improvement over:
```
foo = run(["foo", "bar", "baz"], check=True, text=True, capture_output=True)
```

And the 2nd version hase 2 benefits - its more explicit, and people are more likely to know the standard library API.